dev/io.openliberty.org.jetbrains.kotlin/bnd.bnd

```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props,~../cnf/resources/bnd/transform.props
+
+Bundle-Name: JetBrains Kotlin Jakarta
+Bundle-SymbolicName: io.openliberty.org.jetbrains.kotlin
+Bundle-Description: Liberty JetBrains Kotlin, version ${bVersion}; Jakarta Enabled
+
+bVersion=1.0
+
+Import-Package: \
+  *
+
+Export-Package: \
+  kotlin;version="1.0.0",\
+  kotlin.*;version="1.0.0"
```

Comment:
Versions are still wrong but you can fix that in a follow up PR
```suggestion
  kotlin;version="1.6.20",\
  kotlin.*;version="1.6.20"
```

Reply:
Need to fix the version of `org.jetbrains.kotlin:kotlin-stdlib-common;version=1.6.2` so might as well fix this too